templates/base/navbar.html

```diff
@@ -8,7 +8,31 @@
     <ul class="navbar-nav mr-auto">
       <li class="nav-item active">
         <a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
+      </li><li class="nav-item">
+        <a class="nav-link" href="#">Link</a>
       </li>
+      <li class="nav-item dropdown">
```

Comment:
please move the drop down to the left. Check Bootstrap syntax how to position items to the left and right.